.github/workflows/smoke.yaml

```diff
@@ -35,8 +38,8 @@ jobs:
           distribution: ${{ matrix.clusters.distribution }}
           version: ${{ matrix.clusters.version }}
           kubeconfig: ${{ runner.temp}}/kubeconfig.yaml
-          kubeceptionToken: ${{ secrets.KUBECEPTION_TOKEN }}
-          gkeCredentials: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
+          kubeceptionToken: ${{ matrix.clusters.distribution == 'Kubeception' && env.KUBECEPTION_TOKEN }}
```

Comment:
I'd prefer that the parameter was not set, in case there are issues handling missing parameters, but I think this works for now.

Reply:
Yeah, I just didn't want to redo all of these right now.
Ideally these are split apart with different sets of parameters, but it would take a bit of work to change/test these.